root/static/scripts/edit/components/AddButton.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * @flow strict
+ * Copyright (C) 2024 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+component AddButton(
+  id: string,
+  onClick: (event: SyntheticEvent<HTMLButtonElement>) => void,
+  label?: string,
+) {
+  if (label == null) {
+    return (
+      <button
+        className="add-item"
+        id={id}
+        onClick={onClick}
+        type="button"
+      />
+    );
+  }
+
+  return (
+    <button
+      className="with-label add-item"
+      id={id}
+      onClick={onClick}
+      type="button"
+    >
+      {l(label)}
```

Comment:
This won't work unless the passed-in label is using `N_l` or `N_lp`, though...I see we have:

```
           addButtonLabel => 'Add ISO 3166-1',
```
in at least one case, and
```
           addButtonLabel => lp('Add ISNI code', 'interactive'),
```
in another, neither of which would work with `l()`.

Reply:
Since we're passing translated labels when needed, I just dropped the `l()`